pom.xml

```diff
@@ -1156,6 +1156,12 @@
             <dependency>com.sun.xml.bind:jaxb-core</dependency>
             <dependency>com.sun.xml.bind:jaxb-impl</dependency>
             <dependency>javax.activation:javax.activation-api</dependency>
+            <!-- 
```

Comment:
Please exclude them in the modules where they are used for testing - not globally. In general, we want to be able to detect them accidentally being used so we can remove them in the non-essential cases.